.github/workflows/release.yaml

```diff
@@ -0,0 +1,24 @@
+name: release
+
+on:
+  push:
+    tags:
+      - "v*.*.*"
+
+jobs:
+
+  release:
+    runs-on: ubuntu-20.04
+    steps:
+      - uses: actions/checkout@v3
+      - uses: actions/setup-go@v3
+        with:
+          go-version: 1.17
+      - uses: docker/login-action@v2
+        with:
+          username: ${{ secrets.DOCKERHUB_USER }}
+          password: ${{ secrets.DOCKERHUB_PASSWORD }}
+      - run: echo GIT_TAG=$(echo $GITHUB_REF | awk -F'/' '{ print $NF }') >> $GITHUB_ENV
```

Comment:
I thought we could use `github.ref_name` for that purpose. According to the documentation[1] `ref_name` holds:

```
The branch or tag name that triggered the workflow run.
```

[1] https://docs.github.com/en/actions/learn-github-actions/contexts

Reply:
updated to GITHUB_REF_NAME